addons/base_action_rule/tests/base_action_rule_test.py

```diff
@@ -92,35 +92,33 @@ def test_02_check_from_draft_to_done_with_steps(self):
         verified, the responsible will stay the same
         The responsible in that test will never change
         """
-        cr, uid = self.cr, self.uid
-        filter_draft = self.create_filter_draft(cr, uid)
-        filter_done = self.create_filter_done(cr, uid)
-        self.create_rule(cr, uid, 'on_write', filter_pre_id=filter_draft, filter_id=filter_done)
-        new_lead_id = self.create_lead_test_1(cr, uid)
-        new_lead = self.model.browse(cr, uid, new_lead_id)
+        filter_draft = self.create_filter_draft()
+        filter_done = self.create_filter_done()
+        self.create_rule('on_write', filter_pre_id=filter_draft.id, filter_id=filter_done.id)
+        new_lead = self.create_lead_test_1()
         self.assertEquals(new_lead.state, 'draft')
-        self.assertEquals(new_lead.user_id.id, self.admin)
-        """ change the state of new_lead to open and check that responsible doen't change"""
+        self.assertEquals(new_lead.user_id.id, self.admin_id)
```

Comment:
If this test failed then fail message be include in the arguments like : 
self.assertEquals(arg1, arg2, " message ")